feature-libs/pickup-in-store/root/pickup-in-store-root.module.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { CmsConfig, provideDefaultConfigFactory } from '@spartacus/core';
+import { STORE_FINDER_FEATURE } from '@spartacus/storefinder/root';
+import {
+  CartPickupOptionsContainerModule,
+  OrderConsignmentContainerModule,
+  PdpPickupOptionsContainerModule,
+} from './components/index';
+
+import {
+  PICKUP_IN_STORE_CORE_FEATURE,
+  PICKUP_IN_STORE_FEATURE,
+} from './feature-name';
+
+export function defaultPickupInStoreComponentsConfig(): CmsConfig {
+  return {
+    featureModules: {
+      [PICKUP_IN_STORE_FEATURE]: {
+        cmsComponents: [
+          'CheckoutReviewPickup',
+          'MyPreferredStoreComponent',
+          'OrderConfirmationPickUpComponent',
+          'PickupInStoreDeliveryModeComponent',
+        ],
+        dependencies: [STORE_FINDER_FEATURE],
```

Comment:
In `pickup-in-store-schematics-config.ts`, you have this `dependencyFeatures: [CART_BASE_FEATURE_NAME],`. But why here its dependencies is `STORE_FINDER_FEATURE`?